### ISSUE TITLE
die - All output should be run through an escaping function

### ISSUE THREAD

**Issue by kkmuffme** [WordPress/WordPress-Coding-Standards#1861]
Getting WordPress.Security.EscapeOutput.OutputNotEscaped for following code:

```php
$var = 'hello';
echo $var == 'foo' ? 'bar' : die( 'world' );

echo 'example';
```

while using `die( 'world' );` standalone, does not produce this error.

**Comment by meevly** [comment]
@kkmuffme it's not about `die ( 'world' )`, you get the error because of `echo 'example';` not being escaped (and likely `'bar'` as well). I don't think it's a bug.

**Comment by jrfnl** [comment]
@meevly Sorry, but that's not true. `example` and `bar` are hard-coded strings and don't need to be escaped.

I suspect the problem is with the ternary, but this will need further investigation. If I remember correctly, there are a number of other issues open about ternaries and output escaping.
